validator/validator.protoascii

```diff
@@ -2278,6 +2278,23 @@ tags: {
     supported_layouts: NODISPLAY
   }
 }
+# <amp-embed>
```

Comment:
Yes thank you I did that - this is the (merged) commit:
https://github.com/ampproject/amphtml/commit/679e684f5be2c70a86de1d72d66d659a3bd99b4a